pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -43,6 +44,8 @@ import (
 
 const ResourceResyncTime = 60 * time.Second
 
+const OrphanFinalizerID = "orphanFinalizer"
```

Comment:
Yeah, I could live with that.

On Wed, May 18, 2016 at 5:56 PM, Derek Carr <notifications@github.com>
wrote:

> In pkg/controller/garbagecollector/garbagecollector.go
> <https://github.com/kubernetes/kubernetes/pull/25599#discussion_r63808466>
> :
>
> > @@ -43,6 +44,8 @@ import (
> >
> >  const ResourceResyncTime = 60 * time.Second
> >
> > +const OrphanFinalizerID = "orphanFinalizer"
>
> Custom finalizers should have a prefix assuming finalizers are qualified
> names (like namespaces). So if you write a custom one it needs to be
> mycompany.org/mything. We should reserve the unqualified space for
> Kubernetes. So validation would check that non-standard finalizers were
> qualified. Just like resources....
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25599/files/7b7917ddcb2f2ab0d8c22748df3668346ea69505#r63808466>
>